docs/usage/types/urls.md

```diff
@@ -2,56 +2,50 @@
 
 For URI/URL validation the following types are available:
 
-- `AnyUrl`: any scheme allowed, TLD not required, host required
-- `AnyHttpUrl`: scheme `http` or `https`, TLD not required, host required
-- `HttpUrl`: scheme `http` or `https`, TLD required, host required, max length 2083
-- `FileUrl`: scheme `file`, host not required
-- `PostgresDsn`: user info required, TLD not required, host required,
-  as of V.10 `PostgresDsn` supports multiple hosts. The following schemes are supported:
-  - `postgres`
-  - `postgresql`
-  - `postgresql+asyncpg`
-  - `postgresql+pg8000`
-  - `postgresql+psycopg`
-  - `postgresql+psycopg2`
-  - `postgresql+psycopg2cffi`
-  - `postgresql+py-postgresql`
-  - `postgresql+pygresql`
-- `MySQLDsn`: scheme `mysql`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `mysql`
-  - `mysql+mysqlconnector`
-  - `mysql+aiomysql`
-  - `mysql+asyncmy`
-  - `mysql+mysqldb`
-  - `mysql+pymysql`
-  - `mysql+cymysql`
-  - `mysql+pyodbc`
-- `MariaDBDsn`: scheme `mariadb`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `mariadb`
-  - `mariadb+mariadbconnector`
-  - `mariadb+pymysql`
-- `CockroachDsn`: scheme `cockroachdb`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `cockroachdb+asyncpg`
-  - `cockroachdb+psycopg2`
-- `AmqpDsn`: schema `amqp` or `amqps`, user info not required, TLD not required, host not required
-- `RedisDsn`: scheme `redis` or `rediss`, user info not required, tld not required, host not required (CHANGED: user info) (e.g., `rediss://:pass@localhost`)
-- `MongoDsn` : scheme `mongodb`, user info not required, database name not required, port
-  not required from **v1.6** onwards), user info may be passed without user part (e.g., `mongodb://mongodb0.example.com:27017`)
-- `stricturl`: method with the following keyword arguments:
-    - `strip_whitespace: bool = True`
-    - `min_length: int = 1`
-    - `max_length: int = 2 ** 16`
-    - `tld_required: bool = True`
-    - `host_required: bool = True`
-    - `allowed_schemes: Optional[Set[str]] = None`
-
-!!! warning
-    In V1.10.0 and v1.10.1 `stricturl` also took an optional `quote_plus` argument and URL components were percent
-    encoded in some cases. This feature was removed in v1.10.2, see
-    [#4470](https://github.com/pydantic/pydantic/pull/4470) for explanation and more details.
-
-The above types (which all inherit from `AnyUrl`) will attempt to give descriptive errors when invalid URLs are
-provided:
+- [`AnyUrl`][pydantic.networks.AnyUrl]: any scheme allowed, top-level domain (TLD) not required, host required.
```

Comment:
I wonder if it makes sense to tabulate these and maybe even generate the table programmatically?

Reply:
This is a good callout. I'll look into it today.